$NetBSD: patch-data_com.github.maoschanz.drawing.gschema.xml,v 1.2 2020/11/26 15:26:20 nia Exp $

pkgsrc users are probably more likely to be using a desktop where
server-side decorations are the norm.

--- data/com.github.maoschanz.drawing.gschema.xml.orig	2020-10-31 19:12:11.000000000 +0000
+++ data/com.github.maoschanz.drawing.gschema.xml
@@ -4,7 +4,7 @@
   <schema id="com.github.maoschanz.drawing" path="/com/github/maoschanz/drawing/">
     <!-- Actual options -->
     <key type="s" name="deco-type">
-      <default>''</default>
+      <default>'mtc'</default>
       <summary>Type of window decorations</summary>
       <description>
         Can be '' (automatic), 'hg' (GNOME headerbar), 'he' (elementary OS
